automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/billing/BillingManagementPage.scala

```diff
@@ -84,6 +87,32 @@ class BillingManagementPage(implicit webDriver: WebDriver) extends BaseFireCloud
     val userEmailElement = find(emailQuery)
     userEmail == userEmailElement.get.text
   }
+
+  /**
+    *
+    * @param billingProjectName
+    * @param user
+    * @return Status of "success", "failure", or "unknown"
+    */
+  def newBillingProject(billingProjectName: String, user: Credentials): Option[String] = {
```

Comment:
to be used

Reply:
Nit: Hard to review this out of context of being used. It would be better if this were saved for the PR where it is used.